combine_trios.py

```diff
@@ -2,18 +2,25 @@
 import re
 import os
 import time
-from sys import argv
+import argparse
 import concurrent.futures
 
 startTime = time.time()
 char = '\n' + ('*' * 70) + '\n'
 
-#Input file or list of files
```

Comment:
This file, as far as I can tell, only takes a single input file, so I changed the comment.